eth_worker/eth_manager/blockchain_sync/blockchain_sync.py

```diff
@@ -0,0 +1,156 @@
+import json
+from uuid import uuid4
+import config
+import requests
+from requests.auth import HTTPBasicAuth
+
+from web3 import (
+    Web3,
+    WebsocketProvider,
+    HTTPProvider
+)
+
+import eth_manager.blockchain_sync.blockchain_sync_constants as sync_const
+from eth_manager import red, w3_websocket, persistence_module
+from eth_manager.ABIs import erc20_abi
+
+# Hit the database to get the latest block number to which we're synced
+def get_latest_block_number():
+    return w3_websocket.eth.getBlock('latest').number
+
+# Call app-level webhook with newfound transacitons
+def call_webhook(transaction):
+    body = {
+        'sender_blockchain_address': transaction.sender_address,
+        'recipient_blockchain_address': transaction.recipient_address,
+        'blockchain_transaction_hash': transaction.hash,
+        'transfer_amount': int(transaction.amount),
+        'contract_address': transaction.contract_address
+    }
+    r = requests.post(config.APP_HOST + '/api/v1/credit_transfer/internal/',
+                        json=body,
+                        auth=HTTPBasicAuth(config.INTERNAL_AUTH_USERNAME,
+                                        config.INTERNAL_AUTH_PASSWORD),
+                        timeout=30
+                    )
+    return r
+
+# Get list of filters from redis. This is the starting point of the synchronization process
+def synchronize_third_party_transactions():
+    filters = persistence_module.get_all_synchronization_filters()
+    # Since the webook will timeout if we ask for too many blocks at once, we have to break 
+    # the range we want into chunks. Once all the chunk-jobs are formed and loaded into redis,
+    # then trigger process_all_chunks, which will consume those jobs from redis
+    for f in filters:
+        ceiling = 0
```

Comment:
shouldn't this be `ceiling = latest_block` to prevent the worker from going back to block zero the next time around?